config/consensus.go

```diff
@@ -1383,22 +1383,34 @@ func initConsensusProtocols() {
 	// for the sake of future manual calculations, we'll round that down a bit :
 	v37.ApprovedUpgrades[protocol.ConsensusV38] = 10000
 
-	// ConsensusFuture is used to test features that are implemented
-	// but not yet released in a production protocol version.
-	vFuture := v38
+	v39 := v38
+	v39.ApprovedUpgrades = map[protocol.ConsensusVersion]uint64{}
 
-	vFuture.ApprovedUpgrades = map[protocol.ConsensusVersion]uint64{}
+	v39.LogicSigVersion = 10
+	v39.EnableLogicSigCostPooling = true
+
+	v39.AgreementDeadlineTimeoutPeriod0 = 4 * time.Second
+
+	v39.StateProofBlockHashInLightHeader = true
```

Comment:
nit: move this two lines below to have all three agreement-related options grouped together

Reply:
See latest version